lib/utils/theme_config.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+class ThemeConfig {
```

Comment:
Maybe to keep things organized lets make that AppColors and keep paddings and so on in a separate file?

Reply:
"theme_config" has now:
- AppColors with colors only
- AppBoxProperties with paddings, margins, borders and shadows sizes (empty for now)